_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy_defaults.cpp

```diff
@@ -1350,8 +1350,8 @@ class GetDefault
             ++pRplEnd[lx];
         });
 
-        l0 = std::min<mfxU8>((mfxU8)maxL0, mfxU8(tmpRPL[0] - pRplEnd[0]));
-        l1 = std::min<mfxU8>((mfxU8)maxL1, mfxU8(tmpRPL[1] - pRplEnd[1]));
+        l0 = std::min<mfxU8>((mfxU8)maxL0, mfxU8(pRplEnd[0] - tmpRPL[0]));
```

Comment:
why not just ```l0``` and ```l1```?
```
l0 = std::min(mfxU8(maxL0), l0);
```

Reply:
pRplEnd[0] - tmpRPL[0] != l0. tmpRPL can also include frames not listed in preferred list.
And it is only a bug fix that is easy to understand, btw.